### ISSUE TITLE
Optionally display an image above the question

### ISSUE THREAD

**Issue by teolemon** [shivasurya/FeedbackDialog#5]
We've been considering implementing something like that in Open Food Facts for a while.
So glad your library exists :)

It would be great to be able to display an image above the question, to serve as a reference, to be able to make modals like:

"Does this photo contain the organic label ?"

**Comment by shivasurya** [comment]
Yes, I have an idea of showing images as part of the feedback dialog.
will [this](https://pbs.twimg.com/media/DU4Q7jBU0AAZmqS.jpg) be good? 

let me know you can access the link? and feedback on adding an image to similar one in the screenshot ?

Add your own requirement to make generalize one to make it work for everyone.

Awaiting your reply.

**Comment by shivasurya** [comment]
@teolemon any updates?

**Comment by teolemon** [comment]
looks good @shivasurya :)

**Comment by teolemon** [comment]
I was thinking that our usecase might require having a button to zoom and pan the image to look for details (we have food pictures)
The current implementation is really good though :)

**Comment by teolemon** [comment]
@shivasurya we're implementing your lib in Open Food Facts. As promised, obligatory screenshot :-)
![image](https://user-images.githubusercontent.com/1689815/53405594-76b74a00-39b8-11e9-9e71-2d295e42ecce.png)
Multiple questions (on the roadmap) would be very helpful for us :-)

**Comment by shivasurya** [comment]
Thank you first of all!
Let me check and update here.

**Comment by shivasurya** [comment]
Multiple questions in the sense, can you please elaborate the case and steps here ( UI )

**Comment by teolemon** [comment]
Here's an example on Google Maps.
![image](https://user-images.githubusercontent.com/1689815/54281474-1d474180-459a-11e9-87c7-866721b498fe.png)
![image](https://user-images.githubusercontent.com/1689815/54281521-2fc17b00-459a-11e9-8171-e1829747ac98.png)
